libtestutils/ShortLivingDirectory.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/** @file ShortLivingDirectory.cpp
+ * @author Marek Kotewicz <marek@ethdev.com>
+ * @date 2015
+ */
+
+#include <boost/filesystem.hpp>
+#include "ShortLivingDirectory.h"
+#include "Common.h"
+
+using namespace std;
+using namespace dev;
+using namespace dev::test;
+
+ShortLivingDirectory::ShortLivingDirectory()
+{
+	m_path = getRandomPath();
```

Comment:
I'm using `random_device`, so it's ok. The returned path is using `getDataDir` internally and looks like this : `/~.ethereum/EthereumTests/random_suffix`